src/Sentry.Maui/SentryMauiAppBuilderExtensions.cs

```diff
@@ -99,13 +99,26 @@ private static void RegisterMauiEventsBinder(this MauiAppBuilder builder)
 
                     var platformApplication = application.Delegate as IPlatformApplication;
                     platformApplication?.HandleMauiEvents(bind: false);
+
+                    SentryMauiEventProcessor.InForeground = false;
```

Comment:
Is it true that at this point it's not on the foreground anymore?
If it crashes right after, would be correct to say it wasn't on the foreground?

Reply:
From Apple docs: 

> For apps that support background execution, this method is generally not called when the user quits the app because the app simply moves to the background in that case. However, this method may be called in situations where the app is running in the background (not suspended) and the system needs to terminate it for some reason.

Reply:
worth adding some links to those docs from the code